lib/ansible/modules/monitoring/grafana_dashboard.py

```diff
@@ -141,7 +167,7 @@
 '''
 
 import json
-
+import re
```

Comment:
I think it is now unused, right ?

Reply:
right. with the dedicated `dashboard_id` no need to regex match the path to check if its an id.